packages/notification-center-angular/projects/notification-center-angular/src/lib/notification-center.component.ts

```diff
@@ -15,7 +15,7 @@ customElements.define('notification-center-web-component', NotificationCenterWeb
     [tabs]="tabs"
     [showUserPreferences]="showUserPreferences"
     [allowedNotificationActions]="allowedNotificationActions"
-    [popover]="popover"
+    [popoverConfig]="popoverConfig"
```

Comment:
the angular and vue packages do use the `Web Component` under the hood, that's why their props are changed too